### PR TITLE
D4G-188: Upgrade core from 10.2.8 to 10.3.10 (latest 10.3.x).

### PR DESCRIPTION
**Pull request by maxstarkenburg**
**TL;DR: I did a lot of checking for things that I don't think affect the site or are relevant to it, and I think the update went fine.**

As part of this upgrade, I did some tests:
* Updated a node post-update
* Added a users and logged in as them post-update
* Checked the `dblog` for anything erroring

I also checked the following things given the [release notes for 10.3.0](https://www.drupal.org/project/drupal/releases/10.3.0):
* Grepped modules and theme for any instances of `ImageStyleDownloadController` (none found)
* Grepped modules and theme for any instances of `user/logout` (only results were some contrib tests)
* Ran a DB query (using adminer) for any content instances of `user/logout` (none found in content tables)
* Checked that the toolbar didn't overlap public-themed pages while logged in
* Grepped modules and theme for things like `stickyheader` (no matches), `.sticky-` and `sticky-header` (only matches related to webform Likert scales, which has no config matches on the site)
* `lando composer --version` (already on 2.7.7)
* Checked for any `ckeditor_*` modules (none found)
* Grepped modules and theme for any instances of `jquery.form.js` (none found)

I also checked release notes for 10.3.1-10.3.6 (and later for 10.3.7-10.3.10) for "important" notes, not finding anything new or likely applicable.

Given the config updates resulting from `updb`, I checked:
* Whether the h4 pagination header might merit update any changes on non-core views, but couldn't find that said views (`webinar` and `board_member`) seem to actually have any site usage (they just have block displays and I couldn't find the blocks placed anywhere or referenced in the theme).
* Where/if `layout_builder` is event used. Config matches show up related to Basic Pages and Blog Author nodes, but none of their displays actually have LB in allowed use. So I think we should probably just disable LB (and of course the new core module) until we have intentional plans for it, but that'd be another ticket.

### CODE REVIEW

**Comment by maxstarkenburg** [comment]
FWIW, I tried to "Rerun job" in Acquia (supposedly failed at config:import) but got `Failed to start pipeline: Error message: You do not have permission to run pipelines`.

**Comment by maxstarkenburg** [comment]
For 10.3.7-10.3.10, I did another brief read of "important update information" in their release notes (I don't think any were likely applicable to this site) and did spot checking, including visiting pages, updating a node, and checking the dblog.
